modin/engines/base/frame/data.py

```diff
@@ -963,8 +963,8 @@ def _build_mapreduce_func(self, axis, func):
             A function to be shipped to the partitions to be executed.
         """
 
-        def _map_reduce_func(df):
-            series_result = func(df)
+        def _map_reduce_func(df, *args, **kwargs):
```

Comment:
Why are `args` and `kwargs` needed?

Reply:
to add the ability to function to obtain any arguments, like `other`